content/applications/general/email_communication/email_servers.rst

```diff
@@ -5,122 +5,122 @@ Send and Receive Emails in Odoo with an Email Server
+on :guilabel:`Test Connection`.
 
 Here is a typical configuration for a G Suite server.
 
 .. image:: email_servers/outgoing-server.png
-    :align: center
+   :align: center
+   :alt: The typical G Suite configuration on Odoo.
 
 Then set your email domain name in the General Settings.
 
 .. note::
-   If you get a ``[AUTHENTICATIONFAILED] Invalid credentials (Failure)`` warning when you *Test
-   Connection* on a Gmail address, activate the *Less secure app access* option. A direct link can
-   be `accessed here <https://myaccount.google.com/lesssecureapps?pli=1>`_.
-
-   In addition to that, enable the IMAP setting on your Gmail account.
+   If you get a `[AUTHENTICATIONFAILED] Invalid credentials (Failure)` warning when you
+   :guilabel:`Test Connection` on a Gmail address, activate the :guilabel:`Less secure app access`
+   option. A direct link can be `accessed here <https://myaccount.google.com/
+   lesssecureapps?pli=1>`_.
 
+   In addition to that, enable the :guilabel:`IMAP setting` on your Gmail account.
 
 Can I use an Office 365 server
 ------------------------------
-You can use an Office 365 server if you run Odoo on-premise.
-Office 365 SMTP relays are not compatible with Odoo Online unless you configure
-Odoo to :ref:`force the outgoing "From" address <email_communication/default_from>` .
 
-Please refer to `Microsoft's documentation
-<https://support.office.com/en-us/article/How-to-set-up-a-multifunction-device-or-application-to-send-email-using-Office-365-69f58e99-c550-4274-ad18-c805d654b4c4>`_
-to configure a SMTP relay for your Odoo's IP address.
+You can use an Office 365 server if you run Odoo on-premise. Office 365 SMTP relays are not
+compatible with Odoo Online unless you configure Odoo to :ref:`force the outgoing "From" address
+<email_communication/default_from>`.
+
+Please refer to `Microsoft's documentation <https://support.office.com/en-us/article/How-to-set-up-
+a-multifunction-device-or-application-to-send-email-using-Office-365-69f58e99-c550-4274-ad18-
+c805d654b4c4>`_ to configure a SMTP relay for your Odoo's IP address.
 
 How to use a G Suite server
 ---------------------------
-You can use an G Suite server for any Odoo hosting type.
-To do so you need to setup the SMTP relay service. The configuration steps are explained in 
-`Google documentation <https://support.google.com/a/answer/2956491?hl=en>`__.
+
+You can use an G Suite server for any Odoo hosting type. To do so you need to setup the SMTP relay
+service. The configuration steps are explained in `Google documentation <https://support.google.com
+/a/answer/2956491?hl=en>`__.
 
 Restriction
 -----------
-Please note that port 25 is blocked for security reasons on our SaaS and Odoo.sh platform. Try using
-465, 587, or 2525.
+
+Please note that port 25 is blocked for security reasons on our SaaS and Odoo.sh platform. Try
+using 465, 587, or 2525.
 
 .. _email_communication/default_from:
```

Comment:
Not sure what this is supposed to be?

Reply:
@mivu-odoo this is a [custom anchor](https://www.odoo.com/documentation/15.0/contributing/documentation/rst_cheat_sheet.html#custom-anchors) for internal page linking 😁

Reply:
Oooh got it! I saw a couple more sprinkled throughout the doc, so I figured it was there for a reason. Thank you for teaching me something new 😸